art-Creating_a_Service_for_VM_Provisioning/topics/Provisioning_a_VM_Using_SSUI.adoc

```diff
@@ -0,0 +1,20 @@
+[[provision-vm-using-ssui]]
+== Provisioning a Virtual Machine Using the Self Service User Interface
+
+. Log in to the Self Service User Interface at https://<your_cloudforms_appliance>/self_service.
+. Click the *Service Catalog* tab.
+. Click the service you created.
+. In the *Service and VM Name*:
+.. Enter the service name in *Service Name*.
+.. Enter the virtual machine name in *VM Name*.
+.. Enter the virtual machine description in *VM Description*.
+. In *VM Characteristics*, enter *Number of CPUs* and *VM Memory*.
+. Click *Add to Shopping Cart* and access the shopping cart by clicking its icon on the upper-right corner of the screen.
+. Click *Order*. You have ordered a new provisioning request. You can follow its progress and see additional details of the order request by selecting it from the *Orders* tab.
```

Comment:
In this last step, once the procedure is ended, any additional 'next steps' should follow in a separate paragraph. I wonder about this so I looked it up in the IBM guide (p89, ending a procedure).